ember-toucan-core/src/components/form/wrapper.hbs

```diff
@@ -0,0 +1,15 @@
+{{#if (has-block)}}
+  {{yield
+    (hash
+      Label=(component this.Label)
```

Comment:
I think this will fail for Embroider with optimized settings. Basically you are only allowed to use `component` with static strings, or wrapping the dynamic component reference (`this.Label` could be arbitrary things, that Embroider's static analysis is not able to understand) with `ensure-safe-component`. I had the use the latter in headless...

_But_, I don't think we need `component` at all here, when not passing any curried arguments. Just `Label=this.Label` should also work, not?

Reply:
So I gave the following a shot, which made the contextual component interface happy; however, the named blocks are still throwing glint errors.  I went ahead and added a comment below.

```hbs
{{#if (has-block)}}
  {{yield
    (hash Label=this.Label Hint=this.Hint Control=this.Control Error=this.Error)
  }}
{{else}}
  {{yield this.Label to="label"}}
  {{yield this.Hint to="hint"}}
  {{yield this.Control to="control"}}
  {{yield this.Error to="error"}}
{{/if}}
```

```js
interface ToucanFormWrapperComponentSignature {
  Element: HTMLDivElement;
  Args: {};
  Blocks: {
    // Happy!
    default?: [
      {
        Label: typeof Label;
        Hint: typeof Hint;
        Control: typeof Control;
        Error: typeof Error;
      }
    ];
   // Not happy :( 
    hint?: any;
    control?: any;
    // (property) ToucanFormWrapperComponent.Hint: TemplateOnlyComponent<ToucanFormHintComponentSignature>
    // Argument of type '[TemplateOnlyComponent<ToucanFormHintComponentSignature>]' is not assignable to parameter of type 'TemplateOnlyComponent<ToucanFormLabelComponentSignature>'.
    // Property '[Data]' is missing in type '[TemplateOnlyComponent<ToucanFormHintComponentSignature>]' but required in type 'TemplateOnlyComponent<ToucanFormLabelComponentSignature>'.glint:ts(2345)
    label?: typeof Label;
    error?: any;
  };
}
```

Reply:
Yeah! Ended up doing: `Label=this.Label` instead and it worked like a charm! Thank you for the suggestion.  Also removed the contextual components from `<Field`